app/javascript/channels/rooms_channel.jsx

```diff
@@ -0,0 +1,24 @@
+import consumer from './consumer';
+
+export default function subscribeToRoom(friendlyId) {
+  consumer.subscriptions.create({
+    channel: 'RoomsChannel',
+    friendly_id: friendlyId,
+  }, {
+    connected() {
+      // Called when the subscription is ready for use on the server
+      console.log('connected');
+    },
+
+    disconnected() {
+      // Called when the subscription has been terminated by the server
+      console.log('disconnected');
+    },
+
+    received(joinUrl) {
+      // Called when there's incoming data on the websocket for this channel
+      console.log('received');
+      window.location.replace(joinUrl);
```

Comment:
Redirect to the joinUrl once the broadcast is received

Reply:
Where does it gets joinUrl from?

Reply:
Yeah it's from the RoomsController